.github/workflows/static_checks.yml

```diff
@@ -92,3 +92,18 @@ jobs:
       - name: Bazel lint check
         run: |
           bash /home/runner/work/oppia-android/oppia-android/scripts/buildifier_lint_check.sh $HOME
+
+  script_checks:
+    name: Script Checks
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Bazel
+        uses: abhinavsingh/setup-bazel@v3
```

Comment:
Do we want to be using actions that are created by individuals? @BenHenning

Reply:
We are currently using this action in the unit_tests.yml workflow: https://github.com/oppia/oppia-android/blob/1cc3fc9c9774485219c226c6ccba3e3453513ec5/.github/workflows/unit_tests.yml#L27